bazel/scripts/run_load_tests.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+set -euo pipefail
+
+###############################################################################
+# FUNCTION DECLARATIONS
+###############################################################################
+
+build_load_tests() {
+    echo "Building load tests..."
+    bazel build //lte/gateway/python/load_tests:all
+    echo "Finished building load tests."
+    echo "#############################"
+}
+
+run_load_tests() {
+    echo "Running load tests..."
+    for LOAD_TEST in "${LOAD_TEST_LIST[@]}";
+    do  
+        echo "#############################"
+        echo "Running load test: ${LOAD_TEST}"
+        # shellcheck disable=SC2086
```

Comment:
This is to make sure `LOAD_TEST` can be interpreted as two arguments here without increasing the complexity of the script too much. The check is only disabled for the next line.